src/reynir_correct/pattern.py

```diff
@@ -1108,6 +1163,290 @@ def dir_loc_læsa(self, match: SimpleTree) -> None:
             )
         )
 
+    def mood_sub_ack(self, match: SimpleTree) -> None:
+        """ Indicative mood is used instead of subjunctive 
+            in concessive subclauses """
+        vp = match.first_match("VP > so_fh")
+        so = vp.first_match("so")
+        assert so is not None
+        start, end = so.span
+        variants = list( [ f for f in so.all_variants if not "fh" in f ] )
+        variants.append("vh")
+        suggest = self.get_wordform(so.tidy_text, so.cat, variants, so.lemma)
+        if not suggest:
+            return
+        text = f"Hér skal notaður viðtengingarháttur sagnarinnar '{so.lemma}'"
+        detail = f"Í viðurkenningarsetningum er aðeins viðtengingarháttur tækur, svo sögnina '{so.tidy_text}' skal skrifa '{suggest}'"
+        self._ann.append(
+            Annotation(
+                start=start,
+                end=end,
+                code="P_MOOD_ACK",
+                text=text,
+                detail=detail,
+                original=so.tidy_text,
+                suggest=suggest,
+            )
+        )
+    
+    def mood_sub_rel(self, match: SimpleTree) -> None:
+        vp = match.first_match("VP > so_vh")
+        so = vp.first_match("so")
+        assert so is not None
+        start, end = so.span
+        variants = list( [ f for f in so.all_variants if not "vh" in f] )
```

Comment:
No need for list() around list comprehension; replace condition with `if f != "vh"`